feature_tests/dart/lib/src/BorrowedFields.g.dart

```diff
@@ -43,11 +43,17 @@ final class BorrowedFields {
   static BorrowedFields fromBarAndStrings(Bar bar, String dstr16, String utf8Str) {
     final dstr16Arena = _FinalizedArena();
     final (dstr16Data, dstr16Length) = dstr16._utf16AllocIn(dstr16Arena.arena);
+    final dstr16 = ffi.Struct.create<_SliceUtf16>();
+    dstr16._data = dstr16Data;
+    dstr16._length = dstr16Length;
     final utf8StrArena = _FinalizedArena();
     final (utf8StrData, utf8StrLength) = utf8Str._utf8AllocIn(utf8StrArena.arena);
+    final utf8Str = ffi.Struct.create<_SliceUtf8>();
```

Comment:
I'd push the struct creation into `allocIn`, but I can follow up

Reply:
I'd love that yeah.